packages/os/mark-successful-boot.service

```
@@ -1,15 +1,14 @@
 [Unit]
 Description=Call signpost to mark the boot as successful after all required targets are met.
-After=multi-user.target
+After=migrator.service
+Requires=migrator.service
 # Each service that must start correctly in order for a boot to be successful should be of type "notify"
 # and include "RequiredBy=mark-successful-boot.service" in its [Install] section.
 
 [Service]
-EnvironmentFile=/etc/network/proxy.env
 Type=oneshot
 RemainAfterExit=true
 ExecStart=/bin/signpost mark-successful-boot
-ExecStartPost=-/usr/bin/metricdog send-boot-success
 
 [Install]
-WantedBy=multi-user.target
+WantedBy=preconfigured.target
```

If we don't mark the boot as successful, we could potentially run for a very long time with no issues, and then fail back to a previous release on reboot.

```suggestion
RequiredBy=preconfigured.target
```